src/components/CTable.vue

```diff
@@ -29,19 +29,29 @@
           </tr>
         </thead>
         <tbody>
-          <c-row
-            v-for="m in displayedMessages"
-            :key="m.messageId"
-            :messageId="m.messageId"
-            :actorName="m.actorName"
-            :priority="m.priority"
-            :name="m.name"
-            :args="m.args"
-            :kwargs="m.kwargs"
-            :enqueuedDatetime="m.enqueuedDatetime"
-            :startedDatetime="m.startedDatetime"
-            :endDatetime="m.endDatetime"
-          ></c-row>
+          <template v-for="m in displayedMessages">
+            <c-row
```

Comment:
![image](https://user-images.githubusercontent.com/6851979/82233537-76de9500-9930-11ea-8b7e-02964c8eaf17.png)
expanded row should be white too and have a border at the bottom.